nav2_map_server/README.md

```diff
@@ -62,44 +138,103 @@ occupied_thresh: 0.65
 free_thresh: 0.196
 ```
 
-The Nav2 software retains the map YAML file format from Nav1, but uses the ROS2 parameter
-mechanism to get the name of the YAML file to use. This effectively introduces a
-level of indirection to get the map yaml filename. For example, for a node named 'map_server',
-the parameter file would look like this:
+- 3D Server
+```
+$ map_server_3d map_3d.yaml
+```
+
+For 3D map server YAML consists of two main arguments file name as `image`, and view point as `origin` :
+
+```yaml
+image: testmap.pcd
+origin: [0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0]
+```
+---
+
+- Parameters files
 
+The Navigation2 software retains the map YAML file format from Nav1, but uses the ROS2 parameter
+mechanism to get the name of the YAML file to use. This effectively introduces a level of indirection to get the map yaml filename. For example, for a node named 'map_server', the parameter file would look like this:
+
+```yaml
+# map_server_params_2d.yaml
+map_server_2d:
+    ros__parameters:
+        yaml_filename: "map_2d.yaml"
 ```
-# map_server_params.yaml
-map_server:
+```yaml
+# map_server_params_3d.yaml
+map_server_3d:
     ros__parameters:
-        yaml_filename: "map.yaml"
+        yaml_filename: "map_3d.yaml"
+```
+
+One can invoke the map service executable directly, passing the params file on the command-line, like this:
+
+```shell
+# For 2D server
+$ ros2 run nav2_map_server map_server_2d __params:=map_server_params_2d.yaml
+```
+```shell
+# For 3D server
+$ ros2 run nav2_map_server map_server_3d __params:=map_server_params_3d.yaml
 ```
 
-One can invoke the map service executable directly, passing the params file on the command line,
-like this:
+If needed there services can be launched by using the provided launch files. Here is a list of available lunch scripts:
 
+| LaunchFile | Assiciated executable | Functionality | Type | Map Type
+| :-------------: | :-----------------: | :--------------: | :----: | :---: |
+| map_saver_server_2d.launch.py | main_server_2d.cpp | Map Saver | LifcyleNode | OccupancyGrid
+| map_saver_server_3d.launch.py | main_server_3d.cpp |  Map Saver | LifcyleNode | PointCloud2
+
+```shell
+$ ros2 launch map_saver_server_2d.launch.py
 ```
-$ map_server __params:=map_server_params.yaml
+```shell
+$ ros2 launch map_saver_server_3d.launch.py
 ```
 
 There is also possibility of having multiple map server nodes in a single process, where the parameters file would separate the parameters by node name, like this:
```

Comment:
I think, this should be moved to separate "Multiple Servers" chapter going after "Map Server" and "Map Saver" ones. As it contains usage of both map *server* and map *saver*. this would be reasonable to move.